models/documentlistmodel.cpp

```diff
@@ -60,27 +60,26 @@ DocumentListModel::~DocumentListModel()
 
 QVariant DocumentListModel::data(const QModelIndex& index, int role) const
 {
-    if( !index.isValid() || index.row() < 0 || index.row() >= d->entries.count() )
+    if (!index.isValid() || index.row() < 0 || index.row() >= d->entries.count())
         return QVariant();
     
-    switch( role )
-    {
-        case FileNameRole:
-            return d->entries.at( index.row() ).fileName;
-        case FilePathRole:
-            return d->entries.at( index.row() ).filePath;
-        case FileTypeRole:
-            return QString().append(d->entries.at( index.row() ).fileType).append(d->entries.at( index.row() ).fileRead.toString(Qt::ISODate));
-        case FileSizeRole:
-            return d->entries.at( index.row() ).fileSize;
-        case FileReadRole:
-            return d->entries.at( index.row() ).fileRead;
-        case FileMimeTypeRole:
-            return d->entries.at( index.row() ).mimeType;
-        case FileDocumentClass:
-            return d->entries.at( index.row() ).documentClass;
-        default:
-            break;
+    switch (role) {
+    case FileNameRole:
+        return d->entries.at( index.row() ).fileName;
+    case FilePathRole:
+        return d->entries.at( index.row() ).filePath;
+    case FileTypeRole:
+        return QString().append(d->entries.at( index.row() ).fileType).append(d->entries.at( index.row() ).fileRead.toString(Qt::ISODate));
```

Comment:
Extra spaces in ```at( index.row() )```. Also string builder or arg could be used for building file the file type role.